src/hotspot/share/classfile/systemDictionary.hpp

```diff
@@ -246,21 +241,12 @@ class SystemDictionary : AllStatic {
   static void restore_archived_method_handle_intrinsics() NOT_CDS_RETURN;
 
   // compute java_mirror (java.lang.Class instance) for a type ("I", "[[B", "LFoo;", etc.)
-  // Either the accessing_klass or the CL/PD can be non-null, but not both.
+  // Either the accessing_klass or the CL can be non-null, but not both.
+  // Callee will fill in CL from the accessing klass, if they are needed.
```

Comment:
The two comment lines ("Either … Callee …") should be one line:

```
+ // Callee will fill in CL from the accessing klass, if the CL is needed.
```

Reply:
One line would be too long and the comment doesn't make any sense anyway. The accessing_klass is never null and the callee doesn't do anything with the class loader, ie it doesn't pass it in.   So I deleted the last two comment lines.